nautobot/utilities/forms/fields.py

```diff
@@ -458,3 +458,75 @@ def prepare_value(self, value):
         if value is None:
             return ""
         return json.dumps(value, sort_keys=True, indent=4)
+
+
+class JSONArrayFormField(forms.JSONField):
+    """
+    A FormField counterpart to JSONArrayField.
+    Replicates ArrayFormField's base field validation: Field values are validated as JSON Arrays,
+    and each Array element is validated by |base_field| validators.
+    """
+
+    def __init__(self, base_field, *, delimiter=",", **kwargs):
+        self.base_field = base_field
+        self.delimiter = delimiter
+        super().__init__(**kwargs)
+
+    def clean(self, value):
+        value = super().clean(value)
+        return [self.base_field.clean(val) for val in value]
+
+    def prepare_value(self, value):
+        if isinstance(value, list):
+            return self.delimiter.join(str(self.base_field.prepare_value(v)) for v in value)
+        return value
+
+    def to_python(self, value):
+        if isinstance(value, list):
+            items = value
+        elif value:
+            items = value.split(self.delimiter)
+        else:
+            items = []
+        errors = []
+        values = []
+        for index, item in enumerate(items):
+            try:
+                values.append(self.base_field.to_python(item))
+            except ValidationError as error:
+                errors.append(error)
+        if errors:
+            raise ValidationError(errors)
+        return values
+
+    def validate(self, value):
+        super().validate(value)
+        errors = []
+        for index, item in enumerate(value):
+            try:
+                self.base_field.validate(item)
+            except ValidationError as error:
+                errors.append(error)
+        if errors:
+            raise ValidationError(errors)
+
+    def run_validators(self, value):
+        super().run_validators(value)
+        errors = []
+        for index, item in enumerate(value):
+            try:
+                self.base_field.run_validators(item)
+            except ValidationError as error:
+                errors.append(error)
+        if errors:
+            raise ValidationError(errors)
+
+    def has_changed(self, initial, data):
+        try:
+            value = self.to_python(data)
+        except ValidationError:
+            pass
```

Comment:
Mind adding a comment explaining why we're catching and ignoring this ValidationError?